packages/server/test/integration/conversations.test.ts

```diff
@@ -72,7 +72,9 @@ describe('Conversations', () => {
   test('List conversations by user', async () => {
     const user = await users.create(state.client.id)
     const convo1 = await conversations.create(state.client.id, user.id)
+    await sleep()
```

Comment:
Be careful by adding sleep in tests as this could lead to flaky tests. Also, the second it requires more than 10ms you'll need to up this value.

Reply:
These tests are time bases though. Conversations are sorted by time created, but if they are inserted too fast the time created is the same, so these sleep fix that

Reply:
Ahhh I see my bad!!